src/main/java/com/powsybl/openloadflow/network/AbstractLfBranch.java

```diff
@@ -38,6 +74,19 @@ protected AbstractLfBranch(LfNetwork network, LfBus bus1, LfBus bus2, PiModel pi
         this.piModel = Objects.requireNonNull(piModel);
     }
 
+    protected static List<LfLimit> createSortedLimitsList(CurrentLimits currentLimits, LfBus bus) {
+        LinkedList<LfLimit> sortedLimits = new LinkedList<>();
+        if (currentLimits != null) {
+            double toPerUnit = bus.getNominalV() / PerUnit.SB;
+            for (LoadingLimits.TemporaryLimit temporaryLimit : currentLimits.getTemporaryLimits()) {
+                double valuePerUnit = temporaryLimit.getValue() != Double.MAX_VALUE ? temporaryLimit.getValue() * toPerUnit : Double.MAX_VALUE;
```

Comment:
`Why the default value for limit value is `Double.MAX_VALUE` ?

Reply:
Because Double.NaN is not supported as value in the implementation `TemporaryLimitAdder` of powsybl-iidm-impl. Do you think we should still support a NaN value?